### DISCUSSION TITLE
Employee Directory Using Modern Search

### DISCUSSION THREAD

**Discussion by eshin087** [microsoft-search/pnp-modern-search#1589] [Q&A] [answered]
Hi everyone, I stumbled upon this great tool and trying to create an employee directory. I was wondering if anyone had an existing guide or reference I could follow.

**Comment by FranckyC** [comment, marked as answer]
Hi @eshin087, you can follow this [tutorial](https://aequos-solutions.github.io/modern-data-visualizer/getting_started/tutorials/build_people_directory_sharepoint_search/). Not the same solution, but pretty much the same options.

**Reply by eshin087** [reply]
Thank you so much!! This is exactly what I was looking for, something to get me started and familiar with the Modern Search.
